sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -441,7 +441,7 @@ export class SessionReceiver {
     this._messageSession!.sessionLockedUntilUtc = await this._context.managementClient!.renewSessionLock(
       this.sessionId!
     );
-    return this._messageSession!.sessionLockedUntilUtc;
+    return this._messageSession!.sessionLockedUntilUtc!;
```

Comment:
@ramya-rao-a: Is this product code change supposed to be included in this PR?

Reply:
Yes, its to fix typescript complaining that `this._messageSession!.sessionLockedUntilUtc` is nullable but the return type from that function is not.

Since we assign a value to `this._messageSession!.sessionLockedUntilUtc` right before that `return` statement, we know that it won't be null/undefined, and adding a `!` is a way to tell typescript that.

It has no affect on the generated javascript code which is what gets eventually used by customers